flake.nix

```diff
@@ -76,15 +77,19 @@
           iso_minimal = self.nixosConfigurations.installer_minimal.config.system.build.isoImage;
         };
       };
+    } // flake-utils.lib.eachSystem [ "x86_64-linux" "aarch64-linux" ] (system:
```

Comment:
We could just use `nixpkgs.lib.genAttrs` for this and skip the extra flake input

Reply:
Done in latest version of this PR